package.json

```diff
@@ -47,14 +47,12 @@
     "transactions"
   ],
   "devDependencies": {
-    "@specron/cli": "^0.0.12",
-    "@specron/spec": "^0.0.12",
+    "@0xcert/ethereum-erc721": "^1.10.0",
+    "@specron/cli": "^0.0.17",
+    "@specron/spec": "^0.0.17",
     "lerna": "^2.11.0",
     "ts-node": "^6.1.1",
     "typescript": "^2.9.1",
     "web3": "git+https://github.com/0xcert/web3.js.git#newAbiLib"
-  },
-  "dependencies": {
```

Comment:
This needs to be a dependency since we will be using it in the code itself as well not only in tests.